coral-schema/src/test/java/com/linkedin/coral/schema/avro/ViewToAvroSchemaConverterTests.java

```diff
@@ -406,17 +406,17 @@ public void testLateralViewArrayWithComplexType() {
         TestUtils.loadSchema("testLateralViewArrayWithComplexType-expected.avsc"));
   }
 
-  // Currently coral-hive does not support lateral view on map type and
-  // it throws IllegalStateException while converting it to RelNode
-  @Test(expectedExceptions = IllegalStateException.class)
+  @Test
   public void testLateralViewMap() {
     String viewSql = "CREATE VIEW v AS " + "SELECT bl.Id AS Id_View_Col, t.Col1, t.Col2 " + "FROM baselateralview bl "
         + "LATERAL VIEW explode(bl.Map_Col_String) t as Col1, Col2";
 
     TestUtils.executeCreateViewQuery("default", "v", viewSql);
 
     ViewToAvroSchemaConverter viewToAvroSchemaConverter = ViewToAvroSchemaConverter.create(hiveMetastoreClient);
-    viewToAvroSchemaConverter.toAvroSchema("default", "v");
+    Schema actualSchema = viewToAvroSchemaConverter.toAvroSchema("default", "v");
+
+    Assert.assertEquals(actualSchema.toString(true), TestUtils.loadSchema("testLateralViewMap-expected.avsc"));
```

Comment:
nit: assertEquals(expected, actual)
here & elsewhere in your diff

Reply:
The definition of `assertEquals` is:
```
  static public void assertEquals(String actual, String expected) {
    assertEquals(actual, expected, null);
  }
```
And we used `assertEquals(String actual, String expected)` for all the tests in coral

Reply:
http://junit.sourceforge.net/javadoc/org/junit/Assert.html#assertEquals(long,%20long) ?

Reply:
oh, you are using testng, not junit :-) sorry